### DISCUSSION TITLE
How to remove blue ring?

### DISCUSSION THREAD

**Discussion by benjick** [shadcn-ui/ui#3358] [Q&A]
I'm using `CommandInput` here, it has the exact same classes as when rendered on the shadcn/ui website. But I'm still seeing this blue ring around this input and others. How can I remove it?

<img width="563" alt="Screenshot 2024-04-04 at 15 44 40" src="https://github.com/shadcn-ui/ui/assets/430872/7a458ad1-4243-4a52-8b8f-738aeff2b7fa">

If I check my `components/ui/input.tsx` file I can see the `Input` has the `focus-visible:outline-none` class

**Comment by shrix1** [comment]
its `focus-visible:ring-0` or `focus-visible:ring-transparent`

**Reply by benjick** [reply]
Is there a way to apply it for everything? Like why isn't it shown on the shadcn ui website?

**Reply by rohan-batoninc** [reply]
You can add this to the `input.ts` and `textarea.ts` files that you've downloaded.

**Reply by shrix1** [reply]
@benjick yaa add it it `input.tsx` and `textarea.tsx` file or add it in `global.css`
